radio/src/gui/colorlcd/widget_settings.cpp

```diff
@@ -147,6 +150,7 @@ WidgetSettings::WidgetSettings(Window* parent, Widget* widget) :
             [=](int16_t newValue) {  // setValue
               widget->getOptionValue(optIdx)->unsignedValue =
                   (uint32_t)newValue;
+            SET_DIRTY();
```

Comment:
can you please check the indenttaion for all changes?
The SET_DIRTY call should be on the same level as "widget->"

Reply:
@gagarinlg Thanks for your review Malte. Changed code accordingly.